5.3a_perfect_dish_to_share.py

```diff
@@ -0,0 +1,24 @@
+def perfect_dish_to_share():
+    """
+    return generator for each number if he is perfect to share
+    perfect is if sum of dividers = sum of dishes pieces
+    :return:the numer if he is perfect
+    """
+    num = 1
+    while True:
+        if sum_of_dividers(num) == num:
+            yield num
+        num += 1
+
+
+def sum_of_dividers(num):
+    """
+    check the amount of dividers of a num
```

Comment:
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation. Num is weak as a variable name, also in documentation use full word.